contrib/slime-asdf.el

```diff
@@ -1,4 +1,5 @@
 (require 'slime)
+(eval-when-compile (require 'cl)) ; lexcial-let*
```

Comment:
It would probably be better to avoid cl all together, as the library has been deprecated.